### ISSUE TITLE
[BUG] ts-node@10.9.2" has unmet peer dependency "@types/node@*"

### ISSUE THREAD

**Issue by dschach** [forcedotcom/sfdx-scanner#1324]
I was unable to use `sf plugins update` because of this, which was shown as an error. I could install the specific 3.20.0 version, but I still got a warning.

```
$ sf plugins install @salesforce/sfdx-scanner@3.20.0
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/sfdx-scanner.
Finished digital signature check.
Installing plugin @salesforce/sfdx-scanner@3.20.0... installed v3.20.0
 ›   Warning: "@salesforce/sfdx-scanner > ts-node@10.9.2" has unmet peer dependency "@types/node@*".

These warnings can only be addressed by the owner(s) of @salesforce/sfdx-scanner.
We suggest that you create an issue at https://github.com/forcedotcom/sfdx-scanner/issues and ask the plugin owners to address them.


Successfully installed @salesforce/sfdx-scanner v3.20.0
```

My plugins info:
```
% sf plugins 
@jongpie/sfdx-bummer-plugin 0.0.17 (link) /Users/<me>/Documents/VSCode/apex-rollup/node_modules/@jongpie/sfdx-bummer-plugin
@salesforce/sfdx-scanner 3.19.0 (3.19.0)
dev 2.1.6

Uninstalled JIT Plugins:
community 3.0.10
custom-metadata 3.0.12
devops-center 1.2.5
env 3.0.13
functions 1.22.11
signups 2.0.13
@salesforce/sfdx-plugin-lwc-test 1.1.1
```

**Comment by dschach** [comment]
And now that I try to run
`sf scanner run --engine=”pmd-appexchange” -t sfdx-source --category="Security" -o CodeAnalyzerPmdAppExchange.csv  --format=csv`
I get an error:
```
 ›   Warning: Plugin @salesforce/sfdx-scanner (3.20.0) differs from the version specified by sf (3.19.0)
Error (2): Expected --engine=”pmd-appexchange” to be one of: eslint, eslint-lwc, eslint-typescript, pmd, pmd-appexchange, retire-js, sfge, cpd
See more help with --help
```
But shouldn't the update have also updated the `Config.json` file?

**Comment by jmsfez** [comment]
Same issue today:

$ sf plugins update
 ›   Warning: @salesforce/cli update available from 2.29.5 to 2.30.8.
@salesforce/cli: Updating plugins... done
 ›   Warning: "@salesforce/sfdx-scanner > ts-node@10.9.2" has unmet peer dependency "@types/node@*".
